osquery/tables/networking/etc_hosts.cpp

```diff
@@ -24,6 +24,11 @@
 namespace osquery {
 namespace tables {
 
+#ifndef WIN32
+std::string kEtcHosts = "/etc/hosts";
+#else
+std::string kEtcHosts = "C:\\Windows\\System32\\drivers\\etc\\hosts";
```

Comment:
dynamically grab the c:\windows dir if it's changed. GetWindowsDirectory and PathCombine can help